includes/class-wcpdf-admin.php

```diff
@@ -261,19 +261,15 @@ public function invoice_columns_data( $column ) {
 
 		$invoice = wcpdf_get_invoice( $order );
 
-		if ( empty( $invoice ) ) {
-			return;
-		}
-
 		switch ( $column ) {
 			case 'invoice_number_column':
-				echo $invoice->get_number();
+				$invoice_number = ! empty( $invoice ) && ! empty( $invoice->get_number() ) ? $invoice->get_number() : '';
+				echo $invoice_number;
 				do_action( 'wcpdf_invoice_number_column_end', $order );
 				break;
 			case 'invoice_date_column':
-				if ( ! empty( $date = $invoice->get_date() ) ) {
-					echo $date->date_i18n( wcpdf_date_format( $invoice, 'invoice_date_column' ) );
-				}
+				$invoice_date = ! empty( $invoice ) && ! empty( $invoice->get_date() ) ? $invoice->get_date() : '';
+				echo $invoice_date->date_i18n( wcpdf_date_format( $invoice, 'invoice_date_column' ) );
```

Comment:
I think you need to make this call in the line above, because if `$invoice_date` is empty the `date_i18n` will throw an error.

Reply:
Yes, perfect!